__tests__/app.test.js

```diff
@@ -13,153 +13,320 @@ beforeEach(() => {
-              expect(category.hasOwnProperty("slug")).toBe(true);
-            });
-            expect(res.body.categories.length).toBeGreaterThan(1);
-          });
+test("GET /api/categories : status:200 and response message", () => {
+  return request(app).get("/api/categories").expect(200);
+});
+
+test("GET /api/categories : returns an array of objects with a slug and desc", () => {
+  return request(app)
+    .get("/api/categories")
+    .expect(200)
+    .then((res) => {
+      res.body.categories.forEach((category) => {
+        expect(typeof category).toBe("object");
+        expect(category.hasOwnProperty("description")).toBe(true);
+        expect(category.hasOwnProperty("slug")).toBe(true);
       });
+      expect(res.body.categories.length).toBeGreaterThan(1);
     });
-  });
-  describe("GET /api/reviews", () => {
-    test("status:200 and response message", () => {
-      return request(app).get("/api/reviews").expect(200);
-    });
-    describe("GET /api/reviews", () => {
-      test("returns an array of objects with an owner, title, review_id, category, review_img_url, created_at, votes, designer", () => {
-        return request(app)
-          .get("/api/reviews")
-          .expect(200)
-          .then((res) => {
-            res.body.reviews.forEach((review) => {
-              expect(typeof review).toBe("object");
-              expect(review.hasOwnProperty("owner")).toBe(true);
-              expect(review.hasOwnProperty("title")).toBe(true);
-              expect(review.hasOwnProperty("review_id")).toBe(true);
-              expect(review.hasOwnProperty("category")).toBe(true);
-              expect(review.hasOwnProperty("review_img_url")).toBe(true);
-              expect(review.hasOwnProperty("created_at")).toBe(true);
-              expect(review.hasOwnProperty("votes")).toBe(true);
-              expect(review.hasOwnProperty("designer")).toBe(true);
-            });
-          });
+});
+test("GET /api/users : status:200 and response message", () => {
+  return request(app)
+  .get("/api/users")
+  .expect(200);
+});
+
+test("GET /api/users : returns an array of objects with a username, name, avatar_url", () => {
+  return request(app)
+    .get("/api/users")
+    .expect(200)
+    .then((res) => {
+      res.body.users.forEach((user) => {
+        expect(typeof user).toBe("object");
+        expect(user.hasOwnProperty("username")).toBe(true);
+        expect(user.hasOwnProperty("name")).toBe(true);
+        expect(user.hasOwnProperty("avatar_url")).toBe(true);
       });
-      test("returns total count of all the comments with this review_id", () => {
-        return request(app)
-          .get("/api/reviews")
-          .expect(200)
-          .then((res) => {
-            res.body.reviews.forEach((review) => {
-              expect(review.hasOwnProperty("comment_count")).toBe(true);
-            });
-          });
+      expect(res.body.users.length).toBeGreaterThan(1);
+    });
```

Comment:
I would put expect length assertion above the forEach, just for clarity.

Reply:
Will do thanks!